package.json

```diff
@@ -20,16 +26,36 @@
     "test": "grunt test"
   },
   "devDependencies": {
+    "@dojo/cli": "next",
```

Comment:
Which means you should be able to remove the `typings.json` and use `@types/yargs` instead